libbeat/cmd/instance/beat.go

```diff
@@ -1083,13 +1083,16 @@ func initPaths(cfg *common.Config) error {
 	// the paths field. After we will unpack the complete configuration and keystore reference
 	// will be correctly replaced.
 	partialConfig := struct {
-		Path paths.Path `config:"path"`
+		Path   paths.Path `config:"path"`
+		Hostfs string     `config:"system.hostfs"`
```

Comment:
Is `system.hostfs` kept here for backwards-compatibility?

If we are moving this option from the system module to a general path option, the new option could be called `path.hostfs`, following the pattern of other path options. We would need to keep `system.hostfs` for backwards compatibility in Metricbeat.

Reply:
Yah, this is entirely about backwards compatibility. Should we accept `path.hostfs` and `system.hostfs`?

Reply:
I would suggest to deprecate `system.hostfs` in favour of `path.hostfs` if we are going to start using `hostfs` as other configurable paths. Specially if we make this available in other beats that don't necessarily have a system module.

Reply:
Considering this is a bugfix, should we do that in a separate PR?

Reply:
deprecation would mean that we have to print a log message if we figure out that `system.hostfs` is set. The setting should not be removed yet. I think we can do the log message in this PR already.

Reply:
Yep, log message has been added here.